Code/Java/Jump_Game.java

```diff
@@ -71,39 +71,39 @@ public static int jumpGame(int array[]) {
 	// main method
 	public static void main(String[] args) {
 
-			Scanner sc = new Scanner(System.in);
-			// Number of elements we want to put in array
-			int number = sc.nextInt();
-			int[] array = new int[number];
+		Scanner sc = new Scanner(System.in);
+		// Number of elements we want to put in array
+		int number = sc.nextInt();
+		int[] array = new int[number];
 
-			// Putting values in array using for loop
-			for(int i = 0 ; i < array.length ; i++) {
-				array[i] = sc.nextInt();
-			}
-			// Initializing the value returned by jumpGame method to jumps
-			int jumps = JumpGame.jumpGame(array);
+		// Putting values in array using for loop
+		for(int i = 0 ; i < array.length ; i++) {
+			array[i] = sc.nextInt();
+		}
+		// Initializing the value returned by jumpGame method to jumps
+		int jumps = JumpGame.jumpGame(array);
 
-			System.out.println("minimum number of Jumps required : "+jumps);
+		System.out.println("minimum number of Jumps required : "+jumps);
 	}
 }
 /*
 - Test Cases :
-   1.  Input : 7
-               2 1 3 4 1 2 3
-      Output : minimum number of Jumps required : 3
+ 1.  Input : 7
+             2 1 3 4 1 2 3
+    Output : minimum number of Jumps required : 3
 
-   2.  Input : 4
-               1 0 0 3
-      Output : minimum number of Jumps required : -1
+ 2.  Input : 4
+             1 0 0 3
+    Output : minimum number of Jumps required : -1
 
-   3.  Input : 1
-               0
-      Output : minimum number of Jumps required : 0
+ 3.  Input : 1
+             0
+    Output : minimum number of Jumps required : 0
 
 - Complexity Analysis :
-    * Time Complexity : O(n)
+ * Time Complexity : O(n)
```

Comment:
Specify n not in the form of a para but after writing space and time complexity, you may just mention like
Where n is length of the input array or something similar :)

Reply:
@raksha009  I have a doubt that how can 'n' be specified as length or something else, as when we calculate time complexity we take number of iterations as a function for example f(n) = 5n + 13; and we take the highest degree term where 'n' is just a variable
Please correct me if I am wrong

Thank you